src/components/shared/CallNodeContextMenu.js

```diff
@@ -673,7 +734,7 @@ function TransformMenuItem(props: {|
     <MenuItem
       onClick={props.onClick}
       data={{ type: props.transform }}
-      attributes={{ title: props.title }}
+      attributes={{ title: oneLine`${props.title}` }}
```

Comment:
I added `oneLine` here. We had this for all the titles but after the ftl change, we miss this. So I put it here to cover all of them.

Reply:
What about removing all the `title` from the callers? I find it a bit confusing to still have them there...